### PR TITLE
CLI state between calls 

### LINKED ISSUE #170: CLI state between calls
State needs to persist between CLI calls

### PR DESCRIPTION
**Pull request by tyshko-rostyslav**
Closes #170

### CODE REVIEW

File: rln-cli/src/state.rs [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
 
 #[derive(Default)]
 pub(crate) struct State<'a> {
     pub rln: Option<RLN<'a>>,
 }
+
+impl<'a> State<'a> {
+    pub(crate) fn load_state() -> Result<State<'a>> {
+        let config = Config::load_config()?;
+        let rln = if let Some(InnerConfig { file, tree_height }) = config.inner {
```
Comment (rymnc) [inline rln-cli/src/state.rs:15]: are we also storing the tree_config here?

**Review by curryrasul — APPROVED** [review]
Looks good! ty

File: rln-cli/src/state.rs (hunk 1 above)
Comment (tyshko-rostyslav) [reply rln-cli/src/state.rs:15]: Needed to initialise the `RLN` object

**Review by rymnc — APPROVED** [review]
LGTM

**Review by rahulghangas — APPROVED** [review]

**Pull request merged**